examples/plugins/ebpf/main.go

```diff
@@ -0,0 +1,179 @@
+package main
```

Comment:
Wouldn't it make sense to write this directly in python? If we want _bcc_-people to write plugins it will make it more enticing.